app/src/main/java/com/internship/move/utils/Utils.kt

```diff
@@ -36,4 +38,16 @@ fun interface ClickCallBack {
 
 fun logTag(tag: String, message: String = "") {
     println("[$tag] $message")
-}
\ No newline at end of file
+}
+
+fun checkMail(mailEditText: TextInputEditText): Boolean {
+    return Patterns.EMAIL_ADDRESS.matcher(mailEditText.text.toString()).matches() && mailEditText.text?.length!! > 7
+}
+
+fun checkUserOrPassword(userOrPasswordEditText: TextInputEditText): Boolean {
+    return userOrPasswordEditText.text?.length!! > 4
```

Comment:
Don't use the `!!` operator.
```userOrPasswordEditText.text?.length > 4 ?: false` is a better approach.```.

Also, don't pass in the whole EditText widget, because this way you can't properly reuse the function.
Better, just pass in the string, may it be null or not.